lib/auth/usertoken.go

```diff
@@ -148,7 +148,7 @@ func (a *Server) CreateResetPasswordToken(ctx context.Context, req CreateUserTok
 		return nil, trace.BadParameter("invalid reset password token request type")
 	}
 
-	_, err = a.ResetPassword(ctx, req.Name)
+	err = a.ResetPassword(ctx, req.Name)
 	if err != nil {
```

Comment:
Here and in the other places that you've touched, might as well clean up the code a bit.

```suggestion
	if err := a.ResetPassword(ctx, req.Name); err != nil {
```